### ISSUE TITLE
Inconsistent error state border color

### ISSUE THREAD

**Issue by marksalmon** [livewire/flux#786]
Hi there! 👋

Just a minor thing, but I think the border colors for inputs with the `$invalid` state are different for different types of input:

Inputs have border-red-400
Selects, textareas and editors have border-red-500

livewire/flux-pro/stubs/resources/views/flux/select/button.blade.php (line 29)
```php
->add($invalid
    ? 'border border-red-500'
    : 'border border-zinc-200 border-b-zinc-300/80 dark:border-white/10'
)
```

livewire/flux-pro/stubs/resources/views/flux/input/index.blade.php (line 54)
```php
->add(match ($variant) { // Border...
    'outline' => $invalid ? 'border-red-400' : 'shadow-sm border-zinc-200 border-b-zinc-...',
    'filled'  => $invalid ? 'border-red-400' : 'border-0',
})
```

livewire/flux-pro/stubs/resources/views/flux/textarea.blade.php (line17)
livewire/flux-pro/stubs/resources/views/flux/editor/index.blade.php (line 17)
```php
->add($invalid ? 'border-red-500' : 'border-zinc-200 border-b-zinc-300/80 dark:border-white/10')
```

![Image](https://github.com/user-attachments/assets/397de53c-7c51-4f4b-ad1f-fa09fdeddf38)

**Comment by joshhanley** [comment]
@marksalmon thanks for reporting! I've submitted a PR to fix this.

**Comment by joshhanley** [comment]
This has been released in v1.0.30
